js/tji-datasets-explore.js

```diff
@@ -80,7 +78,7 @@ TJIGroupByBarChart.prototype.create = function(data) {
         }
       ]
     },
-    options: this.get_options()
+    options: this.get_options(grouped.counts)
```

Comment:
Since we are storing `ordered_keys` on the object for a similar purpose (persisting a visual style across filtering/rerendering), maybe we should put this in a variable as well (e.g. `initial_max_count`), instead of passing it around?

Reply:
👍